keps/sig-api-machinery/3488-cel-admission-control/README.md

```diff
@@ -1083,6 +1105,38 @@ repeated evaluations if they are shared across validations.
     - expression: "itemMetadataNames.exists(name, name == 'required')"
 ```
 
+Since the variable names must be valid CEL names, this is checked at creation
+time.
```

Comment:
More detail here?  I would expect:
- Valid CEL variable name according to CEL grammar (regex + keyword collision check)
- Not a reserved Kubernetes variable name (object, oldObject, request, ...)

Reply:
Fixed, I've made this a bit more clear + linked to the CEL language definition.